app/src/main/java/com/codesoom/assignment/exception/ProductNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.codesoom.assignment.exception;
+
+/**
+ * findById로 product를 찾지 못했을 때 던지는 exception
+ */
+public class ProductNotFoundException extends RuntimeException{
```

Comment:
지금은 `findById`를 쓰는 경우에만 이 예외를 던지죠. 이 주석에는 몇 가지 문제가 있습니다.

- `findById`가 아니어도 이 예외를 던지도록 작업하게 될 수 있습니다.
- `findById`의 이름일 바뀌게 될 수 있습니다.
- `findById`가 다른 예외를 던지도록 수정될 수 있습니다.

```suggestion
/**
 * 상품을 찾지 못했을 때 던집니다.
 */
public class ProductNotFoundException extends RuntimeException{
```

Reply:
앗 메소드로 한정지을게아니라 어떤상황에서 발생되는건지를 생각해야겠네요 감사합니다 !

Reply:
"발생"이 아니라 "던진다"라고 생각해 보세요.